accessibility-checker-extension/src/ts/options/OptionsApp.tsx

```diff
@@ -114,7 +116,7 @@ class OptionsApp extends React.Component<{}, OptionsAppState> {
       selected_archive: item.selectedItem,
       rulesets,
       selected_ruleset,
-      show_notif: false
+      show_notif: false,
```

Comment:
Any reason a comma was added here?

Reply:
Strange, VS Code 'Format Document' added that it.
Removed